### ISSUE TITLE
QuestionView.js

### ISSUE THREAD

**Issue by rubendata** [udacity/FSND#108]
Your code in line 130 is wrong and returns an object which is not valid for react.
Please change it to {this.state.categories[id].type} so that your students do not have the trouble with your frontend

**Comment by SudKul** [comment]
Hi @rubendata 
We are not able to find the file you are talking about. Please raise a PR with your proposed updates. 
Thank you.

**Comment by rubendata** [comment]
https://github.com/udacity/FSND/blob/master/projects/02_trivia_api/starter/frontend/src/components/QuestionView.js

**Comment by SudKul** [comment]
Hey @rubendata 
I ran the app with and without your proposed changes at [this line](https://github.com/udacity/FSND/blob/e698dd2331de9cd0e2ec82fa49fe9c8a491f20ff/projects/02_trivia_api/starter/frontend/src/components/QuestionView.js#L130). I did not encounter any issues except the categories are now displayed using the image only. Earlier they were being rendered using category name and image. I believe it's better to display the category name + image, both. 

Besides, I have also checked the Knowledge hub for exact same recommendation from any mentor. There is no such suggestion for the said line of code. 

Such situations do occur in the development environments too, therefore we have the last course - Containerization, that will make the app behave in the same manner on all platforms. 

Thanks
